### PR TITLE
https://github.com/mapgears/ol3-google-maps/issues/46

### PR DESCRIPTION
**Pull request by fipp**
Added olgm.OLGoogleMaps.prototype.getMap

### CODE REVIEW

File: src/ol3googlemaps.js [hunk 1]
```diff
@@ -131,6 +131,15 @@ olgm.OLGoogleMaps.prototype.getGoogleMapsActive = function() {
 
 
 /**
+ * @return {google.maps.Map}
+ * @api
+ */
+olgm.OLGoogleMaps.prototype.getMap = function() {
```
Comment (adube) [inline src/ol3googlemaps.js]: `getMap` might be too generic.  We have 2 maps: GoogleMaps and OpenLayers.  Both have maps.

I suggest naming the method: `getGoogleMapsMap`.  That ways, it's very verbose of what it returns.

**Comment by fipp** [comment]
I updated the function name now @adube, from `getMap` to `getGoogleMapsMap`.

**Comment by adube** [comment]
Thanks.  Would you please merge your commits in a single one ?

**Comment by fipp** [comment]
Like that? Never done exactly this before. Did this on my fork:

```
git reset --soft HEAD~2
git commit -av
git push --force origin master
```

Would it be any better way to do it?

**Comment by adube** [comment]
That could do the trick.  Here's some tips, though:
- you should always create a branch for any fix or feature you want to contribute.  I.e. don't do them in your master branch of your fork.
- the trick I personally do:

```
git log (copy the revision number just before your commit)
git reset --soft the_revision_just_before_your_commit
git commit -m "Add olgm.OLGoogleMaps#getGoogleMapsMap"
git push --force origin yourbranch
```

This would also be useful in the future if you need to rebase your master branch onto the official master one.

**Comment by fipp** [comment]
Good thoughts. Thanks!

**Comment by adube** [comment]
Thanks @fipp for the patch!

**Pull request merged**